index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav id="navcontainer">
+    <div>
+      <h3 id="menulogo">John Doe</h3>
+    </div>
+    <div>
+      <img id="burgermenu" src="./img/burger.png" alt="">
+    </div>
+  </nav>
+
+  <header>
+    <div id="headercontainer">
+      <h1 id="hero">Hey there. I'm Mauricio</h1>
+      <h3 id="subhero">I am a Software Developer</h3>
+      <p id="herotext">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <div>
+        <img src="./img/social/github.png" alt="">
+        <img src="./img/social/linkedin.png" alt="">
+        <img src="./img/social/scissors.png" alt="">
+        <img src="./img/social/twitter.png" alt="">
+        <img src="./img/social/m.png" alt="">
+      </div>
+    </div>
+  </header>
```

Comment:
- [ ] Great job, here instead of a header it'll be more semantic to use a `<section></section>`, kindly fix that.
- [ ] In the design this part has a **background image**, please add the background from the design to your page.

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/164527390-dd3101af-8806-4f9f-a3fe-94e3617be535.png)